test/integration/rule-content-route.yaml.tmpl

```diff
@@ -1,6 +1,9 @@
-destination: "{{.destination}}.{{.namespace}}.svc.cluster.local"
+destination: {{.destination}}.{{.namespace}}.svc.cluster.local
 precedence: 2
 match:
+  source: {{.source}}.{{.namespace}}.svc.cluster.local
+  source_tags:
+    version: v1
   http:
```

Comment:
This needs to go as a separate test to ensure that other services are unable to call service "worldl